ci/ci_steps.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-ENVOY_BUILD_SHA=fc747b3c2fd49b1260484572071fe4194cd6824d
+ENVOY_BUILD_SHA=2867b337182c79ec5229a4261ecdadbf916f49ab4629dfea787cf76c91952dc4
```

Comment:
I think this should be 1d2a2100708d1012cd61054ce04d4dc4e3b03ff2 ?

https://hub.docker.com/r/lyft/envoy-build/tags/